csv.h

```diff
@@ -314,7 +337,7 @@ namespace io{
                         int desired_byte_count;
                 };
         }
-
+		template<class quote_policy = no_quote_escape<','>>
```

Comment:
made LineReader templated so it could call the quote policy functions 
quote_policy::hasQuote();
quote_policy::getQuoteChar();

Reply:
This breaks code using LineReader directly. Previously, people could write io::LineReader my_reader. With this change, they have to write  io::LineReader<> my_reader. This is a breaking change.

Reply:
How about the following then:
change 
template<...>
class io::LineReader
to
template<...>
class io::LineReaderT      (TBD)

and add
typedef LineReaderT<> LineReader

This way the change will not break anything and allow LineReader to find the next line with regards to quoting policy?

Reply:
That is the only option, if LineReader must be changed. A downside of this is that error message get more complex as users are now confronted with a type name they did not use themselves. This is the same situation as std::string vs std::basic_string<char, ...>. 

A different option would be to have two LineReaders and make them an argument to CSVReader